yarn-project/aztec-rpc/src/simulator_oracle/index.ts

```diff
@@ -92,4 +101,27 @@ export class SimulatorOracle implements DBOracle {
       index,
     });
   }
+
+  /**
+   * Retrieve the databases view of the Historic Block Data object.
+   * This structure is fed into the circuits simulator and is used to prove against certain historic roots.
+   *
+   * @returns A Promise that resolves to a ConstantHistoricBlockData object.
+   */
+  getConstantHistoricBlockData(): Promise<ConstantHistoricBlockData> {
```

Comment:
Could you not just use `db.getHistoricBlockData` directly here instead of reimplementing? Also, the naming is not fully consistent if including `constant` or not.

Reply:
correct this is clutter